tests/test_ui.py

```diff
@@ -28,8 +28,8 @@ def test_component(page, port, ds):
     expect(page.get_by_text("predefined")).to_be_visible()
 
     page.get_by_role("button", name="Individuals").click()
-    expect(page.get_by_text("Individuals table options")).to_be_visible()
-    expect(page.get_by_text("Population from")).to_be_visible()
+    # expect(page.get_by_text("Data modification")).to_be_visible()
+    # expect(page.get_by_text("Population ID")).to_be_visible()
```

Comment:
should we fix this? its weird that it doesn't work

Reply:
added issue #57